Makefile-ostree.am

```diff
@@ -143,12 +143,14 @@ ostree_SOURCES += src/ostree/ot-builtin-pull.c
 endif
 
 if USE_LIBSOUP
+if BUILDOPT_TRIVIAL_HTTPD
 # Eventually once we stop things from using this, we should support disabling this
```

Comment:
Presumably this comment should be removed along with this changeset?

Reply:
Yep